examples/skip_plot_map_in_3d.py

```diff
@@ -39,10 +38,12 @@
 plotter.plot_line(line)
 
 # Define a SkyCoord for to set the positon of the camera
-camera_position = SkyCoord(0*u.deg, 0*u.deg, 8*const.R_sun, obstime=m.observer_coordinate.obstime, frame=frames.HeliographicStonyhurst)
+camera_position = m.observer_coordinate
 plotter.set_camera_coordinates(camera_position)
 
 # Rotate the camera by a given angle
 plotter.rotate_camera(45*u.deg)
+# Set the view angle of the plot
```

Comment:
What does a view angle mean in this context?

Reply:
Could be interpreted as the field of view of the mesh (in our case all the map)

Reply:
Fair enough, I think this should be explained more in the example.

Reply:
yeah, do you think it would be better with a separate example where we demonstrate setting of camera coordinate and view angle?

Reply:
If that example can be made sufficaly complex, sure.

Reply:
Hmm, I was thinking setting camera coordinate, linking camera properties from pyvista's documentation and setting of view angle can be passed off as complex enough?

Reply:
Have to see the final version but it could be.

Reply:
I think having a basic "here is a 3D plot" and a more involved "how the camera works" example would be good.

Reply:
Sure, I'll add all of them to a separate example